video/out/drm_atomic.h

```diff
@@ -70,6 +71,16 @@ struct drm_object {
     drmModePropertyRes **props_info;
 };
 
+#include "config.h"
```

Comment:
includes go at the top
empty structs will throw a warning, you need something like `char dummy;` in an `#else`